apps/platform/src/app/auth/page.tsx

```diff
@@ -51,6 +52,49 @@ export default function AuthPage(): React.JSX.Element {
     }
   }
 
+  const handleGithubLogin = (): void => {
+    setIsLoading(true)
+    window.location.href = `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/auth/github`
+  }
+
+  const handleGithubRedirect = async () => {
+    const urlParams = new URLSearchParams(window.location.search)
+    const code = urlParams.get('code')
+
+    if (code) {
+      try {
+        const response = await fetch(
+          `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/auth/github/callback?code=${code}`,
+          {
+            method: 'GET',
+            credentials: 'include'
+          }
+        )
+
+        if (!response.ok) {
+          throw new Error('Network response was not ok')
+        }
+
+        const data = (await response.json()) as User
+        setEmail(data.email)
+
+        if (response.status === 200) {
+          router.push('/auth/account-details')
+        }
+      } catch (error) {
+        // eslint-disable-next-line no-console -- we need to log the error
+        console.error(`Failed to fetch user data: ${error}`)
+      }
+    } else {
+      // eslint-disable-next-line no-console -- we need to log the error
+      console.error('No authorization code found in the URL')
```

Comment:
Add toast here too. Go through the Figma file to know the color scheme.

Reply:
got it :+1: